src/inmanta/module.py

```diff
@@ -260,10 +258,9 @@ def __init__(self, msg: str, validation_error: Optional[ValidationError] = None)
 
     @classmethod
     def _extend_msg_with_validation_information(cls, msg: str, validation_error: ValidationError) -> str:
-        for error in validation_error.errors():
-            mgs: str = error["msg"]
-            error_type = error["type"]
-            msg += f"\n{error['loc']}\n\t{mgs} ({error_type})"
+        errors = validation_error.errors()
+        if errors:
+            msg += "\n" + display_errors(errors)
```

Comment:
```suggestion
            msg += "\n" + textwrap.indent(display_errors(errors), " " * 2)
```
I think this could improve readability a bit.